examples_tests/01.HelloWorld/main.cpp

```diff
@@ -15,13 +15,19 @@ nothing fancy, just to show that Irrlicht links fine
 #include <volk/volk.h>
 #include "../../src/nbl/video/CVulkanPhysicalDevice.h"
 #include "../../include/nbl/video/surface/ISurfaceVK.h"
+#include "../../src/nbl/video/CVulkanImage.h"
+#include "../../src/nbl/video/CVulkanSemaphore.h"
+#include "../../src/nbl/video/CVulkanFence.h"
 
 #include <nbl/ui/CWindowManagerWin32.h>
-#include <nbl/system/ISystem.h>
 #include "../common/CommonAPI.h"
 
 using namespace nbl;
 
+// This probably a TODO for @sadiuk
+static bool windowShouldClose_Global = false;

```

Comment:
put it inside `DemoEventCallback` (or just copy the one from `CommonAPI` and put it here)

Reply:
resolved